hazelcast-client-legacy/src/test/java/com/hazelcast/client/topic/ClientReliableTopicTest.java

```diff
@@ -67,13 +79,13 @@ public void testCreation() {
 
     @Test(expected = NullPointerException.class)
     public void addMessageListener_whenNull() {
-        ITopic topic = client.getReliableTopic("foo");
+        ITopic topic = client.getReliableTopic(randomString());
```

Comment:
I tend to use random names for the data put to hazelcast or in hazelcast data structures. The reason with this for me to differentiate which value is needed to be exact or which value does not matter what it is. When I see random name, I directly deduce that it does not matter which partition it lands to etc . Think of it as `_` in Haskell.
But random number if iterations or random sleep, they can lead to when trying to reproduce the issue back. This is close to idea of Fuzzy testing, but we are missing `the reporting the random used values` part to use them again when reproducing the test.(There is also finding the minimal failing test in Fuzzy test, but this is another story)